.drone.yml

```diff
@@ -56,6 +57,16 @@ pipeline:
       - ${SBT_RUN}
       - ./bin/ci-clean-cache.sh
 
+  build_windows:
```

Comment:
This is currently a Drone CI step, I wonder if it would be better to run it in parallel. http://docs.drone.io/pipelines/

Reply:
This runs concurrently with the `build` step, is that what you wanted?